### ISSUE TITLE
[Host] High Performance Logging

### ISSUE THREAD

**Issue by zarusz** [zarusz/SlimMessageBus#335]
Currently, most log statements use the logging extension methods to log information. We could use the following to avoid the logging overhead and make it faster:

- For `netstandard2.0` use https://learn.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging
- For `net6.0` onwards use https://learn.microsoft.com/en-us/dotnet/core/extensions/logger-message-generator

**Comment by dundich** [comment]
if not to change anything

https://github.com/stbychkov/AutoLoggerMessage

**Comment by zarusz** [comment]
I don't think it will work if as target netstandard2.0 and net6.0, there's a call out:

> As this solution is based on interceptors, only .NET 8+ is supported
